lib/special_route_publisher.rb

```diff
@@ -0,0 +1,58 @@
+require 'gds_api/publishing_api/special_route_publisher'
+
+class SpecialRoutePublisher
+  def initialize(publisher_options)
+    @publisher = GdsApi::PublishingApi::SpecialRoutePublisher.new(publisher_options)
+  end
+
+  def publish(route)
+    @publisher.publish(
+      route.merge(
+        format: "special_route",
+        public_updated_at: Time.now.iso8601,
+        update_type: "major",
+      )
+    )
+  end
+
+  def routes
+    [
+      {
+        publishing_app: "finder-frontend",
```

Comment:
Is the publishing app technically rummager? I'm not quite sure what "publishing app" means for a special route.